server/routes/calendar.js

```diff
@@ -2,7 +2,42 @@ var express = require("express");
 var router = express.Router();
 
 var pool = require("../db/db");
-
+router.post("/changecoverage", (req, res) => {
+  var coverage = req.body.coverage;
+  approve = true;
+  var shiftID = req.body.shiftID;
+  var notes = req.body.sentNotes;
+  pool.query(
+    "UPDATE shifts SET cover_requested = $1 WHERE sle_id = $2",
```

Comment:
This query will update every shift for the sle with that sle_id, not just the one that they requested to be covered.

Reply:
resolved